### ISSUE TITLE
Suggestion: Handle the case where the caret is placed arbitrarily within a logical soft-tab

### ISSUE THREAD

**Issue by stephenmartindale** [Microsoft/vscode-backspace#2]
Backspace++: v.0.0.12
VS Code: v.0.10.11
Try this with a tab width of `n = 4`
1. Create a line with `2 x n = 8` spaces - a perfect multiple of the tab width.
2. Place the caret after `(2 x n) - 1 = 7` of the spaces - leaving one space to the _right_
3. Press backspace.

I absolutely _love_ the fact that it removes only `n - 1 = 3` characters to the left of the caret, leaving one whole tab. I think it might be awesome if it also removed the space to the right of the caret since it logically belonged to the soft-tab that was removed.

Backspace++ is a life-saver. Thanks for making it!

**Comment by jrieken** [comment]
I like, I wonder if it should be the default tho....

**Comment by Tyriar** [comment]
It would feel strange if spaces to the right of the cursor were deleted on <kbd>backspace</kbd>.

**Comment by jrieken** [comment]
yeah, maybe we implement similar logic for the `deleteRight` command
